### DISCUSSION TITLE
Pricing

### DISCUSSION THREAD

**Discussion by ahoiroman** [lucky-media/cloud#3] [Ideas]
Hello :-) 

I'd love to use this starter kit - but there's one thing: Is there any chance you may add a pricing-component?

Cheers :-)

**Comment by lokmanm** [comment]
Hey @ahoiroman, first of all thanks for the great suggestion. I will pass it to the designer and we will try to ship this by the end of the week.

**Reply by lokmanm** [reply]
Hey @ahoiroman, pricing section was added with the latest release of [v1.2](https://github.com/lucky-media/cloud/releases/tag/v1.2)

Check the commit for the differences and feel free to include it in your project :)

**Comment by ahoiroman** [comment]
Thanks a lot. I will test it :-)
